editor/sidebar/post-format/index.js

```diff
@@ -2,13 +2,13 @@
  * External dependencies
  */
 import { connect } from 'react-redux';
-import { find } from 'lodash';
```

Comment:
I'm actually a bit confused by the behavior if I'm honest. I'd wanted to define HoCs from left-to-right, which is not the functional pattern, and is what I understand [flow](https://lodash.com/docs/4.17.4#flow) to achieve. But in practice this was being reversed, and the props added from the left-most HoC (`connect`) weren't reaching `withAPIData`.

So, I agree on the behavior of defining HoCs, but the use of flow appears to be backwards. I might toy with this a bit to see why it's this way.